modules/end-user-guide/partials/proc_adding-language-support-plug-in-to-the-che-workspace.adoc

```diff
@@ -5,51 +5,58 @@
 [id="adding-language-support-plug-in-to-the-{prod-id-short}-workspace_{context}"]
 = Adding a language support plug-in to a {prod-short} workspace
 
-This procedure describes adding a tool to an existing workspace by enabling a dedicated plug-in from the Dashboard.
+This procedure describes adding a tool to a created workspace by enabling a dedicated plug-in from the Dashboard.
 
-To add tools that are available as plug-ins into a {prod-short} workspace, use one of the following methods:
+//To add tools that are available as plug-ins into a {prod-short} workspace, use one of the following methods:
+//The sentence on the prev line seems to be out of place at the time of writing this comment, hence commenting it out. max-cx
 
 * xref:installing-the-plug-in-by-adding-content-to-the-devfile_{context}[Edit the workspace devfile from the Dashboard *Devfile* tab.]
 
-This procedure uses the Language Support for Java plug-in as an example.
+This procedure uses the Language Support for Java plug-ins as an example.
 
 .Prerequisites
 
-* A running instance of {prod-short}. To install an instance of {prod-short}, see {link-installing-an-instance}.
-
-* An existing workspace defined in this instance of {prod}; see:
-** xref:configuring-a-workspace-with-dashboard.adoc[]
-** xref:creating-a-workspace-from-code-sample.adoc#creating-a-workspace-from-quick-add-view-of-user-dashboard_{context}[Creating a workspace from Quick Add view of user dashboard]
-
- * The workspace must be in a **stopped** state. To stop a workspace:
-
-.. Navigate to the {prod-short} Dashboard. See xref:navigating-che-using-the-dashboard.adoc[].
+* A running instance of {prod-short}.
++
+To install an instance of {prod-short}, see {link-installing-an-instance}.
 
+* A created workspace that is defined in this instance of {prod}
```

Comment:
missing dot

Reply:
:) On purpose. IBM Style Guide: p. 89 "_Write lists so that all or none of the items start with complete sentences. ... If list items comprise only sentence fragments, do not include any end punctuation_."